includes/lib/fastimage/class-fastimage.php

```diff
@@ -70,7 +70,14 @@ public function getType()
                 case chr(0x89).'P':
                     return $this->type = 'png';
                 default:
-                    return false;
+                    $this->strpos = 0;
```

Comment:
Just a clarifying question: why can't the SVG part be on its own 'case' branch rather than on the 'default' one?